src/Billing/EDI270.php

```diff
@@ -741,28 +742,6 @@ public static function eligibilityVerificationSave($subscriber = [])
         }
     }
 
-// return array of X12 partners
```

Comment:
if there is worry about breaking things (ie. other places that may call this function), then could just keep a wrapper here:
```
    public static function getX12Partner($id = 0)
    {
        EDI270EligibilityPartnerService::getX12Partner($id);
    }
```